src/Parser/ShadowPriest/Modules/Spells/Voidform.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Tab from 'Main/Tab';
+
+import { formatPercentage, formatNumber } from 'common/format';
+
+import Mindbender from './Mindbender';
+import Dispersion from './Dispersion';
+import VoidTorrent from './VoidTorrent';
+import VoidformsTab from './VoidformsTab';
+
+
+
+class Voidform extends Module {
```

Comment:
This looks pretty daunting to try to understand. I can't help shake the feeling that some parts may become hard to maintain after a while. I'd suggest refactoring it a bit if possible to make it easier to follow and maintain in the future.

Reply:
Yes, I plan on cleaning this up a lot for the next PR.